### ISSUE TITLE
"yarn.lock" unintentionally replaces "shufo/php-parser" with "czosel/php-parser"

### ISSUE THREAD

**Issue by yaegassy** [shufo/blade-formatter#275]
## Description

Is there any way to solve this problem?

### prepare (Node v14.x.x)

```
mkdir -p /tmp/bfcheck
cd /tmp/bfcheck
npm init -y

yarn add blade-formatter@1.11.4
cp yarn.lock before.lock

yarn add blade-formatter@1.11.5
```

### diff

```
$ diff yarn.lock before.lock
19c19
<     php-parser czosel/php-parser#b55446b48a4a588e8b62c7f9085ce0e7cb3b70fa
---
>     php-parser shufo/php-parser#0b7781fdf88be4b8974320cf27af1588c07e2da8
55,58c55,58
< blade-formatter@1.11.5:
<   version "1.11.5"
<   resolved "https://registry.yarnpkg.com/blade-formatter/-/blade-formatter-1.11.5.tgz#a3b4175603f4c038ca39d013825532c435a83c65"
<   integrity sha512-BtRTC2mbCw0etWomcUHkefqAzInzOYO7FoqyhwQrat9+2PYhF0qcnqjG37UPJggFW0ohqHXsPXwnDYJp7mquhw==
---
> blade-formatter@1.11.4:
>   version "1.11.4"
>   resolved "https://registry.yarnpkg.com/blade-formatter/-/blade-formatter-1.11.4.tgz#089ab1899199a53ae1d79c390049b11aaa5468ce"
>   integrity sha512-Y6ncCuim7YJt2NDQnlKYZzpXwrE4sFjZEUcaHuRSSY5gyxI3SrtyMOlM6Oh7UFAmD7xptRLe5lHIC7el9hNbMA==
```

## Misc

In coc-blade-formatter, I manually modify yarn.lock every time as a workaround.

- REF(e.g.): https://github.com/yaegassy/coc-blade-formatter/commit/1cc54d4aa282637907037ba5454f77c6d9b706ee

## Related issue?

- https://github.com/shufo/blade-formatter/issues/235

**Comment by yaegassy** [comment]
This issue was resolved in v1.11.10. 🙇

**Comment by shufo** [comment]
Thanks @yaegassy.
I have switched to using upstream repository `czosel/php-parser` in `v1.11.10`, and that might solved the issue.
